config/PropertyNames.xml

```diff
@@ -545,33 +545,22 @@ generated from the section label.
     <section name="IceSSL">
         <property name="Alias" />
         <property name="CAs"/>
-        <property name="CertAuthDir" deprecatedBy="IceSSL.CAs"/>
-        <property name="CertAuthFile" deprecatedBy="IceSSL.CAs"/>
         <property name="CertStore" />
         <property name="CertStoreLocation" />
         <property name="CertFile" />
-        <property name="CertVerifier" />
         <property name="CheckCertName" />
         <property name="CheckCRL" />
         <property name="Ciphers" />
         <property name="CertificateRevocationListFiles" />
         <property name="DefaultDir" />
-        <property name="DH.[any]" />
-        <property name="DHParams" />
-        <property name="EntropyDaemon" />
         <property name="FindCert" />
-        <property name="FindCert.[any]" deprecated="true"/>
-        <property name="InitOpenSSL" />
```

Comment:
There was nothing left in the initialization block after removing the other properties, so I removed this one too.